python/fusion_engine_client/messages/sta5635.py

```diff
@@ -84,3 +84,44 @@ def __repr__(self):
         result = super().__repr__()[:-1]
         result += f', seq={self.command_sequence_number}, data={self.data}]'
         return result
+
+
+class STA5635IQData(MessagePayload):
+    """!
+    @brief Wrapper for IQ Samples from a STA5635.
+    """
+    MESSAGE_TYPE = MessageType.STA5635_IQ_DATA
+    MESSAGE_VERSION = 0
+
+    Construct = Struct(
+        Padding(4),
+        # NOTE: Since this message does no capture the expected data size, the Construct relies on the size of the
+        # Python buffer passed to `unpack`` to infer the size of the data. This is the behavior of @ref GreedyBytes.
+        "data" / GreedyBytes
```

Comment:
You need to add imports for `Padding`, `GreedyBytes`, and anything else that's used in this class but not the rest of this file. This class isn't included in the core library, so I think it doesn't get imported by the unit tests. You might want to quickly hack an app or a unit test or something to import this file so you can make sure it has what it needs.